tests/scripts/vars.env

```diff
@@ -0,0 +1,12 @@
+VM_NAME=ngf-auto-testing
```

Comment:
I'm worried that someone might accidentally push this file filled out. Can it live outside the repo in a specified location, like `~/.ngf/vars.env ` or something?

Reply:
I've renamed the example to `vars.env-example` and added all dotenv files to gitignore